mk_verificator/tests/fixtures/base.py

```diff
@@ -9,6 +11,24 @@ def local_salt_client():
 
 
 @pytest.fixture
+def nova_client():
+    config = utils.get_configuration(__file__)
```

Comment:
are you sure that it's working? because there in no config files within fixtures directory